README.Rmd

```diff
@@ -42,22 +40,21 @@ devtools::install_github("svalvaro/drawCell")
 
 ## Examples
 
-
 It requires the [taxonomy id](https://www.ncbi.nlm.nih.gov/taxonomy/) of your
 species of interest, and one or multiple SL codes for subcellular locations 
 that will be colored. The SL codes for each subcellular location can be found at
 [Uniprot](https://www.uniprot.org/docs/subcell) and
 [uniprotkb_sl2go](http://current.geneontology.org/ontology/external2go/uniprotkb_sl2go).
 
-
-
 To generate a cell of Homo sapiens (Taxonomy id: `9606`) with 
 highlighted Endoplasmic Reticulum (SL code: `0095`) and lipid droplets 
 (SL code: `0154`):
 
 ```{r animal_cell,  results='hide', message=F, warning=FALSE, error=F}
+
 library(drawCell)
-drawCell(taxonomy_id = '9606', sl_ids = c('0095','0154'), color = 'yellow')
+
+drawCell(organism_identifier = '9060', list_sl_colors = list("SL0173" = "red", "SL0101" = "blue"))
```

Comment:
We are using different SL than the ones in the explanation. Let's keep the same ones "SL0095" AND "SL0154"